jobsdb/migration.go

```diff
@@ -206,6 +252,11 @@ func (jd *HandleT) cleanupStatusTables(ctx context.Context, dsList []dataSetT) e
 	if err != nil {
 		return err
 	}
+	// vaccum status table if total size exceeds vaccumFullStatusTableThreshold in the toCompact list
+	toVaccum, err := jd.getVacuumCandidates(ctx, toCompact)
```

Comment:
A table that we know is going to be vacuumed full doesn't need to be vacuumed if it is in the cleanup list